src/CMakeLists.txt

```diff
@@ -129,30 +133,24 @@ endif()
 # ==============================================================================
 INCLUDE(CXX11)
 check_for_cxx11_compiler(CXX11_COMPILER)
-# If a C++11 compiler is available, then set the appropriate flags
-IF(CXX11_COMPILER)
-  ADD_DEFINITIONS(-DOPENMVG_USE_CXX11)
-  enable_cxx11()
-  INCLUDE(CheckIncludeFileCXX)
-  CHECK_INCLUDE_FILE_CXX(thread HAVE_CXX11_THREAD)
-  IF (HAVE_CXX11_THREAD)
-    ADD_DEFINITIONS(-DHAVE_CXX11_THREAD)
-  ENDIF(HAVE_CXX11_THREAD)
-    CHECK_INCLUDE_FILE_CXX(chrono HAVE_CXX11_CHRONO)
-  IF (HAVE_CXX11_CHRONO)
-    ADD_DEFINITIONS(-DHAVE_CXX11_CHRONO)
-  ENDIF(HAVE_CXX11_CHRONO)
-ENDIF(CXX11_COMPILER)
+
+if(NOT CXX11_COMPILER)
+  message(FATAL_ERROR "The compiler does not support the CXX11 standard.")
+endif(NOT CXX11_COMPILER)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # ==============================================================================
 # OpenMP detection
 # ==============================================================================
+set(OPENMVG_USE_OPENMP 0)
 IF (OpenMVG_USE_OPENMP)
   FIND_PACKAGE(OpenMP)
   IF (OPENMP_FOUND)
```

Comment:
Yes and indeed the very same it is declared at the beginning, so the ones inside the for can be removed. 
Also, but this could be another PR, I'd really prefer that if an option set by the user is not satisfied the cmake will fatal error rather than silently fallback to `OFF`. What do u think?